test/AppStream.DurablePatterns.Tests/DurablePatternsExecutorTests.cs

```diff
@@ -63,7 +67,7 @@ public void ExecuteAsync_WithFailedActivityStep_ThrowsPatternActivityFailedExcep
                 null, TimeSpan.Zero, steps[0].StepId, steps[0].StepType, false, new Exception());
             stepExecutorMock
                 .Setup(m => m.ExecuteStepAsync(
-                    steps[0], context, null))
+                    steps[0], stepsConfigEntityId, context, null))
```

Comment:
I read StepsConfigEntityId everywhere and it seems like a mouthful... Maybe flowInfoId indeed would be better?